src/programs/raster/mdimtranslate.rs

```diff
@@ -0,0 +1,266 @@
+use crate::{
+    errors::*,
+    utils::{_last_null_pointer_err, _path_to_c_string},
+    Dataset,
+};
+use gdal_sys::{GDALMultiDimTranslate, GDALMultiDimTranslateOptions};
+use libc::{c_char, c_int};
+use std::{
+    borrow::Borrow,
+    cell::Cell,
+    ffi::CString,
+    mem::ManuallyDrop,
+    path::Path,
+    ptr::{null, null_mut},
+};
+
+/// Wraps a [GDALMultiDimTranslateOptions] object.
+///
+/// [GDALMultiDimTranslateOptions]: https://gdal.org/api/gdal_utils.html#_CPPv428GDALMultiDimTranslateOptions
+///
+pub struct MultiDimTranslateOption {
```

Comment:
Should probably be called `MultiDimTranslateOptions`, since there can be multiple options inside.

Reply:
[23957df](https://github.com/georust/gdal/pull/289/commits/23957df6195fc5557422ffd81aa724c7aa4aaaf1)